js/nodes/materials/material_writer.js

```diff
@@ -1,7 +1,7 @@
 class MaterialWriter {
     constructor()
     {
-        this.material_file_name = this.addWidget("string","Materials Filenae", "Materials");
+        this.material_file_name = this.addWidget("string","Materials Filenae", "Materials.json");
```

Comment:
```suggestion
        this.material_file_name = this.addWidget("string","Materials Filename", "Materials.json");
```
Typo

Reply:
I think you are commenting on an outdated file, as the specific file *material_writer.js* has been modified in this PR.